<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IlsLlzPanel</class>
 <widget class="QWidget" name="IlsLlzPanel">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>140</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>ILS/LLZ BRA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameInputs">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblNavaidLayer">
        <property name="text">
         <string>Navaid layer (point)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cboNavaidLayer"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblRoutingLayer">
        <property name="text">
         <string>Routing / runway (line)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cboRoutingLayer"/>
      </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblSiteElev">
           <property name="text">
            <string>Site elevation (m)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="spnSiteElev">
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnDirection">
     <property name="text">
      <string>Forward</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnCalculate">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
     <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
            <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
         </property>
        </spacer>
     </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>